components/02-elements/rating/Rating.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    class="rating"
```

Comment:
What's the difference between this component and `RatingRateItem`?

Reply:
This component shows only average.